### DISCUSSION TITLE
Link to keybinds section of wiki appears broken

### DISCUSSION THREAD

**Discussion by aadamson** [nvim-neorg/neorg#313] [Q&A]
I'm trying to find a web listing of the default neorg keybindings for reference. The link provided on the wiki (https://github.com/nvim-neorg/neorg/wiki/User-Keybinds) and elsewhere appears to redirect to the wiki landing page. What's the correct link? Alternatively, is there a source file I could reference?

**Comment by esquires** [comment]
Here is a link to the current keybinds:

https://github.com/nvim-neorg/neorg/blob/main/lua/neorg/modules/core/keybinds/default_keybinds.lua
